src/components/Delete.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchComputers } from '../redux/computers/computers';
+import ItemDelete from './itemDelete';
+import '../App.css';
+
+const Delete = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchComputers());
+  }, []);
+  const item = [];
```

Comment:
- [ ] **(OPTIONAL)** This is just a naming convention that is a good practice, but since there are multiple things inside of this array, it would be better to use `items` in the plural.